### PR TITLE
Added documentation for reset, reset_code, and update. Updated some imagery in the unity package.

### PR DESCRIPTION
**Pull request by worth12**
May be some grammar mistakes, but I tried to resolve any I could find.

### CODE REVIEW

**Review by worth12 — COMMENTED** [review]
Also, added general tab section to unity database manager section.

File: docs/foundry/database-server/api.md [hunk 1]
```diff
@@ -175,7 +175,51 @@ Authorization Bearer <session_token>
...
+
+#### Response
+```json
+{
+  "message": string, // A message indicating the result of the operation
+}
+```
 
 ### Verify Email
 `// TODO`
```
Comment (WireWhiz) [inline docs/foundry/database-server/api.md]: Looks like we still have a TODO here, let's get that taken care of

**Review by WireWhiz — CHANGES_REQUESTED** [review]
Looking really good, looks like there's one TODO that slipped through the cracks. Also looks like there's a broken link in `/docs/foundry/database-server/overview` to `/docs/foundry/unity-packages/database/overview`, I'm not seeing that in your changes so I think I was the one that broke that, but let's get it fixed so that CI runs.

**Comment by worth12** [comment]
Both are taken care of.

**Pull request merged**